src/internal/animate.ts

```diff
@@ -45,10 +45,29 @@ export function stopAnimations(el: HTMLElement) {
   return Promise.all(
     el.getAnimations().map(animation => {
       return new Promise(resolve => {
-        const handleAnimationEvent = requestAnimationFrame(resolve);
+        const requestId = requestAnimationFrame(timestamp => {
+          animation.oncancel = null;
```

Comment:
What's the purpose of setting these properties to `null`?

Reply:
Here I am trying to remove the event listeners for we no longer need to run their handlers since we're resolving the promise. (in the case where this callback is called before the event handlers);